game-app/game-core/src/main/resources/games/strategy/engine/xml/game.dtd

```diff
@@ -103,7 +103,7 @@
 	>
 	<!-- if no player given, then delegate is initialized with a null player -->
 	<!-- max run count is the maximum number of times that a delegate can run, if specified must be > 0, if not specified its infinity-->
-	<!ELEMENT sequence (step+)>
+	<!ELEMENT sequence (step+ forEach*)>
```

Comment:
I'm surprised we still have the `game.dtd` file around even, is this update needed? Does the map have an unnecessary DTD definition on it?

Reply:
I was surprised too - but I noticed when I was adding a test for this with a map XML that IntelliJ was actually validating against that and giving me red underlines, so I updated it since it wasn't too hard. The game itself doesn't seem to use the DTD for validation (OTOH I think if the editor I was using for editing my map did use it, it could have been helpful).